### ISSUE TITLE
Create loss function(s)

### ISSUE THREAD

**Issue by shaws34** [shaws34/Audio-Undistortion#6]
Create a function to calculate the loss between the ideal output and the actual output.  Right now, I'm think this should compare the output to the original audio clip, but there may be other audio calculations that may be used instead or as a hybrid.  One function may be signal-to-noise ratio, or maybe there should be a way to allow humans to score whether they are pleased with the undistortion.

The downside of the comparison between original input and neural network output is that the original file would have to be considered subjectively perfect in order to train the neural network correctly.  It's possible, though, that the original audio file may be enhanced/improved but the neural network's loss function would train away from the improvement.
